Agents/OSMAgent/README.md

```diff
@@ -3,43 +3,41 @@
 The OSMAgent is an agent that works with OpenStreetMap (OSM) data to link them to existing building IRI and instantiate the semantic representation of building usage information from OSM data.
 The workflow of the agent can be broadly outlined in the following steps:
 1) Categorize OSM tags according to [OntoBuiltEnvironment](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_Ontology/ontology/ontobuiltenv) concept. 
-2) Identify and match OSM data with the 3D buildings uploaded as CityGML data and LoD0 footprint as extracted by the [DataIntegrationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/dev-1524-integration-data-2/Agents/DataIntegrationAgent). To assign building IRI to OSM data, this is carried out through matching the geometry of the OSM data to the 3D buildings' footprint. 
+2) Identify and match OSM data with the 3D buildings uploaded as CityGML data and LoD0 footprint. This is performed by assigning building IRI to OSM data through matching the geometry of the OSM data to the 3D buildings' footprint. 
 3) Calculate building usage share for all OSM data with tagged building IRI and non-null usage information.
 4) If land use data is available, for 3D buildings without tagged OSM usage, the agent will tag it with the corresponding land use.  
```

Comment:
In addition to this broad outline, I think it would be really useful to give some details (either here or in section 4.3 below) about what the agent actually does, i.e. a few sentences on what its output is and where to find that. Does it instantiate any triples? Does it just add columns to existing OSM data tables? Does it create new tables? If so, what's in these columns/tables/triples? I guess the above enumeration already indicates what the output is - it would be nice to know where to find it.

Reply:
Resolved by https://github.com/cambridge-cares/TheWorldAvatar/pull/992/commits/6a0d519788ba8ea3a84589f74f3fc5e95ab44b86